test/csuite/wt4803_cache_overflow_abort/main.c

```diff
@@ -0,0 +1,177 @@
+#include "test_util.h"
```

Comment:
One thing I just noticed, this needs the IP comment at the beginning. See `wt4156_metadata_salvage/main.c` and copy/paste that into here.

Reply:
98d8d9d